index.html

```diff
@@ -0,0 +1,506 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="dist/output.css">
+  <title>stripe</title>
+</head>
+
+<body class="font-open_sans ">
+  <header class="relative px-8  z-50 w-full flex justify-between items-center gap-44 py-3">
+    <img class="md:ml-36 cursor-pointer" src="./assets/images/logo.svg" alt="stripe logo">
+    <nav class="hidden md:flex text-white md:mt-2">
+      <a href="#" class="px-5  text-sm font-bold hover:text-slate-200 cursor-pointer">Products</a>
+      <a href="#" class="px-5  text-sm font-bold hover:text-slate-200 cursor-pointer">Solutions</a>
+      <a href="#" class="px-5  text-sm font-bold hover:text-slate-200 cursor-pointer">Developers</a>
+      <a href="#" class="px-5  text-sm font-bold hover:text-slate-200 cursor-pointer">Resources</a>
+      <a href="#" class="px-5  text-sm font-bold hover:text-slate-200 cursor-pointer">Pricing</a>
+    </nav>
+    <div class="hidden lg:block px-5 py-1 rounded-full bg-zinc-400 hover:font-light">
+      <a href="#" class="text-white  font-bold ">Sign in</a>
+
+    </div>
+    <div class="lg:hidden w-6 relative">
+      <img class=" " src="./assets/images/menu.svg" alt="">
+    </div>
+  </header>
+
+
+
+  <main>
+    <section
+      class="absolute bg-gradient-to-r from-purple-500 via-yellow-400 to-pink-500  ease-in-out duration-1.8s  h-[70vh] w-full  -skew-y-12  -top-48">
+    </section>
+
+    <section class="flex">
+      <div class="flex flex-col gap-9 mt-24 relative px-8 lg:pl-36 md:w-[44rem] md:mt-48 md:ml-36">
+
+        <h1 class="text-6xl tracking-tighter font-bold w-full md:text-7xl md:w-[480px]">Financial infrastructure for the
+          internet
+        </h1>
+        <p class="max-w-[540px] text-lg">Millions of companies of all sizes—from startups to Fortune 500s—use Stripe’s
+          software
+          and APIs to accept
+          payments, send
+          payouts, and manage their businesses online.</p>
+        <div class="space-x-4">
+          <a href="#" class="text-black bg-gray-600 px-5 py-2 rounded-3xl font-semibold">Start now</a>
+          <a href="#" class="font-semibold">Contact sales</a>
+        </div>
```

Comment:
Use the white color and add ending arrows inside the buttons to make them almost as the original ones: 

**Yours**
![image](https://user-images.githubusercontent.com/58233753/203083650-343d03e7-bb61-4aae-9df5-39b8fdcf45d6.png)

**Original**
![image](https://user-images.githubusercontent.com/58233753/203083564-a9b0afba-59cd-49d7-8cfd-4cc5febc45a6.png)